src/App.tsx

```diff
@@ -1,19 +1,139 @@
+  return todosCopy.map(todo => ({
+    ...todo,
+    user: users.find(user => user.id === todo.userId),
+  }));
+};
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+type State = {
+  preparedTodos: Todo[],
+  employee: number,
+  task: string,
+  errorEployee: boolean,
+  errorTask: boolean,
 };
 
+class App extends React.Component<{}, State> {
+  state: State = {
+    preparedTodos: prepareTodos(),
+    employee: 0,
+    task: '',
+    errorEployee: false,
+    errorTask: false,
+  };
+
+  fakeSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    this.setState(prevState => {
+      if (!prevState.employee || !prevState.task) {
+        return {
+          ...prevState,
+          errorEployee: true,
+          errorTask: true,
+        };
+      }
+
+      const maxId = Math.max(...prevState.preparedTodos.map(todo => todo.id));
+
+      todosCopy.push({
+        id: maxId + 1,
+        title: prevState.task,
+        userId: prevState.employee,
+        completed: false,
+      });
+
+      return {
+        employee: 0,
+        task: '',
+        errorEployee: false,
+        errorTask: false,
+        preparedTodos: prepareTodos(),
+      };
+    });
+  };
+
+  employeeValueChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    this.setState({
+      employee: +event.target.value,
+      errorEployee: false,
+    });
+  };
+
+  taskValueChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({
+      task: event.target.value,
+      errorTask: false,
+    });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <h1>Boss simulator</h1>
+        <form
+          className="form"
+          onSubmit={this.fakeSubmit}
+        >
+          <p>
+            Give some tasks for employees
+          </p>
+
+          <select
+            name="employee"
+            id="employee"
+            value={this.state.employee}
+            onChange={this.employeeValueChange}
+          >
+            <option
+              value="0"
+              disabled
+            >
+              Choose lazy employee
+            </option>
+            {users.map(user => (
+              <option
+                value={user.id}
+                key={user.id}
+              >
+                {user.name}
+              </option>
+            ))}
+          </select>
+
+          <p className="error">
+            {this.state.errorEployee && 'You need to pick somebody!'}
+          </p>
+
+          <input
+            type="text"
+            name="task"
+            id="task"
+            placeholder="Give task to a lazy employee"
+            value={this.state.task}
+            onChange={this.taskValueChange}
+          />
+
+          <p className="error">
+            {this.state.errorEployee && 'You need to give task!'}
```

Comment:
change for task error, cause now errors are shown for both inputs
![image](https://user-images.githubusercontent.com/57598825/138253047-a03a1236-2346-40e7-a873-9b371fd03a3f.png)


```suggestion
            {this.state.errorTask && 'You need to give task!'}
```